PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -2420,33 +2420,34 @@ public bool ScrollWidthTracking
         /// <summary>
         /// Adds a new keys to ignore
         /// </summary> 
-        public virtual void AddIgnoredKeys(Keys keys)
+        public virtual void AddIgnoredKeys(ShortcutKeys keys)
         {
-            ignoredKeys.Add((int)keys, (int)keys);
+            //ignoredKeys.Add((int)keys, (int)keys);
```

Comment:
Yep. Since [this](https://github.com/fdorg/flashdevelop/pull/1245/files#diff-05f30fbc757ea5380e769c4a72b326e9L5323) is removed, `ignoredKeys` and `ignoreAllKeys` have no function at all.